cms/static/js/views/course_video_settings.js

```diff
@@ -0,0 +1,545 @@
+
+            if (providerPlan) {
+                $providerEl.empty();
+                HtmlUtils.setHtml(
+                    $providerEl,
+                    HtmlUtils.interpolateHtml(
+                        HtmlUtils.HTML('<input type="radio" id="transcript-provider-none" name="transcript-provider" value="" {checked}/><label for="transcript-provider-none">{text}</label>'),
+                        {
+                            text: gettext('None'),
+                            checked: self.selectedProvider === '' ? 'checked' : ''
+                        }
+                    )
+                );
+
+                _.each(providerPlan, function (providerObject, key) {
+                    var checked = self.selectedProvider === key ? 'checked' : '';
+                    HtmlUtils.append(
+                        $providerEl,
+                        HtmlUtils.interpolateHtml(
+                            HtmlUtils.HTML('<input type="radio" id="transcript-provider-{value}" name="transcript-provider" value="{value}" {checked}/><label for="transcript-provider-{value}">{text}'),
+                            {
+                                text: providerObject.display_name,
+                                value: key,
+                                checked: checked
+                            }
+                        )
+                    )
+                });
+            }
+        },
+
+        populateTurnaround: function() {
+            var self = this,
+                turnaroundPlan = self.getTurnaroundPlan(),
+                $turnaroundContainer = self.$el.find('.transcript-turnaround-wrapper'),
+                $turnaround = self.$el.find('#transcript-turnaround');
+
+            // Clear error state if present any.
+            this.clearPreferanceErrorState($turnaroundContainer);
+
+            if(self.selectedProvider && turnaroundPlan) {
+                $turnaround.empty().append(new Option(gettext('Select turnaround'), ''));
+                _.each(turnaroundPlan, function (value, key) {
+                    var option = new Option(value, key);
+                    if (self.selectedTurnaroundPlan === key) {
+                        option.selected = true;
+                    }
+                    $turnaround.append(option);
+                });
+                $turnaroundContainer.show();
+            } else {
+                $turnaroundContainer.hide();
+            }
+        },
+
+        populateFidelity: function() {
+            var self = this,
+                fidelityPlan = self.getFidelityPlan(),
+                $fidelityContainer = self.$el.find('.transcript-fidelity-wrapper'),
+                $fidelity = self.$el.find('#transcript-fidelity');
+
+            // Clear error state if present any.
+            this.clearPreferanceErrorState($fidelityContainer);
+
+            // Fidelity dropdown
+            if (self.selectedProvider &&fidelityPlan) {
```

Comment:
`if(fidelityPlan)` is enough as `fidelityPlan ` is already checking `self.selectedProvider`